code/codegen/src/main/kotlin/com/android/gradle/replicator/codegen/ClassModel.kt

```diff
@@ -0,0 +1,26 @@
+package com.android.gradle.replicator.codegen
+
+/**
+ * Simple model for generated class.
+ */
+data class ClassModel(
```

Comment:
data classes should not be mutable. any reason you need it to be a data class?

Reply:
no reason, made it a plain class.